auk-notebook-example.ipynb

```diff
@@ -189,12 +190,13 @@
     "    return ret\n",
     "\n",
     "def get_text(by=\"all\", minline=MINIMUM_WORD_LENGTH):\n",
-    "    \"\"\"Get the text from the files (by domain or year if desired)\n",
+    "    \"\"\"Get the text from the files (by domain or year if desired).\n",
     "    \n",
-    "    :param by: \"all\", \"domain\" or \"year\" the output to return\n",
-    "    :param minline: the minimum size of a line to be included in the output\n",
+    "    :param by: \"all\", \"domain\" or \"year\" the output to return.\n",
+    "    :param minline: the minimum size of a line to be included in the output.\n",
     "    :return: [(year/domain, text)] or [text] depending on by\n",
```

Comment:
Maybe `:return: [({year or domain}, textString)] if `by` is 'domain' or 'year', otherwise [textString]\n` is more clear - it is also possible to include the type using :rtype: but I don't think that is too necessary.